docs/pages/components/popover.mdx

```diff
@@ -0,0 +1,112 @@
+export { getStaticProps } from '../../getStaticProps'
+
+import {
+  component,
+  dependencies,
+  name,
+  peerDependencies,
+  version
+} from '@welcome-ui/popover/package.json'
+
+# Popover
+
+<tagversion version={version} name={name} />
+
+Popover from [Reakit](https://reakit.io/docs/popover/) with a really nice theme 👀
+
+## Install and import
+
+<usage component={component} name={name} peerDependencies={peerDependencies} />
+
+## Example
+
+Use `usePopoverState`, `Popover` and `Popover.Trigger` to create a simple Popover.
+
+```jsx
+function() {
+  const popover = usePopoverState()
+
+  return (
+    <>
+      <Popover.Trigger as={Button} {...popover}>
+        Open Popover
+      </Popover.Trigger>
+      <Popover {...popover} withCloseButton>
+          Praesent sit amet quam ac velit faucibus dapibus. Quisque sapien ligula, rutrum quis
+          aliquam nec, convallis sit amet erat. Mauris auctor blandit porta.
+      </Popover>
+    </>
+  )
+}
+```
+
+## Title
+
+Use `Popover.Title` to create a predefined title block.
```

Comment:
Do you mean the `title` prop?
Because you're not using Popover.Title 😛

Reply:
oopsy 🙊